src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,296 @@
     background-color: rgba($brand-02, 0.1);
   }
 }
+
+@mixin data-table-v2-core--x {
+  //================================================
+  //MAIN CONTAINER
+  //================================================
+  .#{$prefix}--data-table-v2-container {
+    min-width: rem(600px);
+    padding-top: $spacing-3xs; // allow space for focus styles
+
+    .#{$prefix}--search-input {
+      background-color: transparent;
+    }
+  }
+
+  //================================================
+  //TABLE HEADER
+  //================================================
+  .#{$prefix}--data-table-v2-header {
+    background: $ui-01;
+    height: rem(88px);
+    color: $text-01;
+    @include font-size('20');
+    font-weight: 400;
+    padding: 22px 0 0 $spacing-md;
+    position: relative;
+    top: rem(21px);
+  }
+
+  .#{$prefix}--data-table-v2-optional {
+    position: relative;
+    left: rem(17px);
+    top: rem(-20px);
+    @include font-size('14');
+    color: $text-02;
+  }
+
+  //================================================
+  //DATA TABLE - DEFAULT STYLES
+  //================================================
+  .#{$prefix}--data-table-v2 {
+    border-collapse: collapse;
+    border-spacing: 0;
+    width: 100%;
+
+    [class*='col'] {
+      outline: none;
+    }
+
+    thead {
+      @include type-style('body-short-01');
+      background-color: $ui-03;
+      font-weight: 700;
+    }
+
+    th:hover {
+      background: $data-table-column-hover;
+    }
+
+    th.#{$prefix}--table-column-icon:hover,
+    th.#{$prefix}--table-column-menu:hover,
+    th.#{$prefix}--table-expand-v2:hover {
+      background: $ui-03;
+    }
+
+    tbody {
+      @include type-style('body-short-01');
+      background-color: $ui-01;
+      width: 100%;
+    }
+
+    tbody tr,
+    thead tr {
+      border: none;
+    }
+
+    tbody tr td {
+      background: $ui-01;
+      border-bottom: 1px solid $ui-03;
+      border-top: 1px solid $ui-01;
+    }
+
+    tbody tr:hover:not([disabled]) {
+      background: $hover-field;
+
+      &:first-of-type {
+        outline: 1px solid $hover-field;
+        outline-offset: -1px;
+      }
+
+      & td {
+        color: $text-01;
+        background: $hover-field;
+        border-bottom: 1px solid $hover-field;
+        border-top: 1px solid $hover-field;
+      }
+    }
+
+    tbody tr:focus {
+      outline: 0;
+    }
+
+    tr {
+      height: rem(48px);
+      width: 100%;
+
+      .#{$prefix}--overflow-menu {
+        opacity: 1;
+      }
+    }
+
+    th,
+    td {
+      @include typescale('zeta');
+      padding-left: $spacing-sm;
+      padding-right: $spacing-sm;
+      vertical-align: top;
+      text-align: left;
+      color: $text-02;
+      border-top: 1px solid $ui-01;
+      border-bottom: 1px solid $ui-01;
+
+      &:first-of-type:not(.#{$prefix}--table-column-checkbox) {
```

Comment:
Is this one more relevant for the selection PR?

Reply:
This would be the padding for if the first column is not a checkbox column, that's why I included it here.

Reply:
What does the corresponding UI for this situation look like?